Sources/Validators/CreditCardValidator.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  CreditCardValidator.swift
+//  FormValidatorSwift
+//
+//  Created by Onur Ersel on 2016-11-02.
+//  Copyright © 2016 ustwo. All rights reserved.
+//
+
+import Foundation
+
+/**
+ *  The `CreditCardValidator` contains an `CreditCardCondition`. A valid string is a credit card number.
+ *  - seealso: `CreditCardCondition`
+ */
+public struct CreditCardValidator: Validator {
+    
+    
+    // MARK: - Properties
+    
+    public var conditions: [Condition]
+    
+    
+    // MARK: - Initializers
+    
+    /**
+     Initializes a `CreditCardValidator` with all card types.
+     */
+    public init() {
+        self.init(cardType: .all)
+    }
+    
+    /**
+     Initializes a `CreditCardValidator`.
+     - parameter cardType: Credit card type.
+     */
+    public init(cardType: CreditCardType) {
+        conditions = [CreditCardCondition(cardType: cardType)]
+    }
+    
+    
+    /**
+     Returns card types which are valid with given number.
+    
+     - Parameters:
+      - creditCardNumber: Credit card number.
+      - completion: Completion callback, returns valid card types for given number.
+     */
+    public func cardTypeOf (creditCardNumber: String, completion : @escaping (CreditCardType) -> Void) {
```

Comment:
It'd be good to also add a note in the header comments that this function is asynchronous. While it should be obvious given the completion handler, it's also nice to add it to the notes as well.